app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -387,7 +206,7 @@ private fun FavoriteParkingCard(parking: Parking, index: Int, onRemove: () -> Un
               onClick = {
                 onRemove()
                 showConfirmDialog = false
-                Toast.makeText(context, "Removed from Favorites!", Toast.LENGTH_SHORT).show()
+                Toast.makeText(context, onFavParkingRemovalText, Toast.LENGTH_SHORT).show()
```

Comment:
This is quite unrelated to your PR per se, but I think we could drop this toast. The user can see that the favorite parking is gone, thus making this toast a bit overkill

Reply:
Thanks for the suggestion! Code coverage :arrow_up: :fire: